ee/frontend/mobile-replay/transformers.ts

```diff
@@ -0,0 +1,269 @@
+import { EventType, fullSnapshotEvent, metaEvent } from '@rrweb/types'
+
+import {
+    fullSnapshotEvent as MobileFullSnapshotEvent,
+    metaEvent as MobileMetaEvent,
+    NodeType,
+    serializedNodeWithId,
+    wireframe,
+    wireframeDiv,
+    wireframeImage,
+    wireframeRectangle,
+    wireframeText,
+} from './mobile.types'
+import { makePositionStyles, makeStylesString, makeSvgBorder } from './wireframeStyle'
+
+/**
+ * generates a sequence of ids
+ * from 100 to 9,999,999
+ * the transformer reserves ids in the range 0 to 9,999,999
+ * we reserve a range of ids because we need nodes to have stable ids across snapshots
+ * in order for incremental snapshots to work
+ * some mobile elements have to be wrapped in other elements in order to be styled correctly
+ * which means the web version of a mobile replay will use ids that don't exist in the mobile replay,
+ * and we need to ensure they don't clash
+ * -----
+ * id is typed as a number in rrweb
+ * and there's a few places in their code where rrweb uses a check for `id === -1` to bail out of processing
+ * so, it's safest to assume that id is expected to be a positive integer
+ */
+function* ids(): Generator<number> {
```

Comment:
Should we just use the `Math.random` builtin instead?

Reply:
I think ids make more sense as a sequence and there's no chance of a clash this way (albeit there's a very small chance of a clash with a clamped random number)